packages/schematics/component/files/__name@dasherize@if-flat__/__name@dasherize__.__type@dasherize__.spec.ts.template

```diff
@@ -8,9 +8,10 @@ describe('<%= classify(name) %><%= classify(type) %>', () => {
   let fixture: ComponentFixture<<%= classify(name) %><%= classify(type) %>>;
 
   beforeEach(waitForAsync(() => {
-    TestBed.configureTestingModule({
+    TestBed.configureTestingModule({<% if(!standalone) {%>
       declarations: [ <%= classify(name) %><%= classify(type) %> ],
-      imports: [IonicModule.forRoot()]
+      imports: [IonicModule.forRoot()]<%} %><% if(standalone) {%>
+      imports: [<%= classify(name) %><%= classify(type) %>],<%} %>
```

Comment:
If standalone we need to pass the component in `imports`. If not standalone we need to pass the component in `declarations` and `IonicModule` in imports.